gatsby_fp/src/components/fieldComponents/profService.js

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+
+export default function ProfService ({profServiceList}){
+  return(
+    
+    <>
+    <div className="sectiontitle">Professional Service</div>
+        <div>
+          <ul>
+          {profServiceList.map((element, index) => (
+            // <li key={index} dangerouslySetInnerHTML ={{__html: 
+            //   `${element.type} - ${element.type_other}<br>${element.elecapp}<br>${element.title} 
+            //   ${element.org} in ${element.city}, ${element.state} (${element.dtm_start} ${element.dty_start} - 
+            //   ${element.dtm_end} ${element.dty_end}) <br> Compensated: 
+            //   ${element.compensated}
+            //   <br>Audience: ${element.audience}`}}>
+            // </li>
+              <li key={index} dangerouslySetInnerHTML ={{__html:` 
```

Comment:
We should avoid using indices as keys. (https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)